gcn/notices/neutrino_telescope/NeutrinoAlert.schema.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/neutrino_telescope/NeutrinoAlert.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "title": "NeutrinoAlert",
+  "description": "Neutrino alert information common to all neutrino telescope alerts",
+  "allOf": [
+    {
+      "$ref": "../core/Event.schema.json",
+      "description": "Unique per-alert identification information from source telescope"
+    },
+    {
+      "$ref": "../core/Alert.schema.json",
+      "description": "Alert information (alert_datetime and alert_type)"
```

Comment:
We don't typically list which fields are included.

Reply:
removed